src/modules/UI/scenes/Scan/Scan.ui.js

```diff
@@ -135,9 +136,9 @@ export default class Scan extends Component<Props> {
 
   renderCamera = () => {
     if (this.props.cameraPermission === AUTHORIZED) {
-      const torchMode = this.props.torchEnabled ? Camera.constants.TorchMode.on : Camera.constants.TorchMode.off
+      const torchMode = this.props.torchEnabled ? RNCamera.Constants.FlashMode.on : RNCamera.Constants.FlashMode.off
 
-      return <Camera style={styles.preview} ref="cameraCapture" torchMode={torchMode} onBarCodeRead={this.onBarCodeRead} />
+      return <RNCamera style={styles.preview} type={RNCamera.Constants.Type.back} ref="cameraCapture" flashMode={torchMode} onBarCodeRead={this.onBarCodeRead} />
```

Comment:
why did this change from torch to flash?

Reply:
Because it's a different library now, and that's what the new library called it.